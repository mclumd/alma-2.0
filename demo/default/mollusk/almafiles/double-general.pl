rel(naked_nautilus, barry).
fif(now(4), rel(cephalopod, steve)).
