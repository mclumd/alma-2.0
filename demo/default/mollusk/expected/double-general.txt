rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
~rel(shell_bearer, steve)
rel(naked_nautilus, barry)
rel(nautilus, barry)
rel(cephalopod, barry)
rel(mollusk, barry)
abnormal(barry, mollusk, "~rel(shell_bearer, barry)")
abnormal(barry, nautilus, "~rel(shell_bearer, barry)")
~rel(shell_bearer, barry)
